src/components/generic/ThemeButton/ThemeButton.scss

```diff
@@ -0,0 +1,92 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}--join-button {
+  background: var(--wxc-join-button-bg);
+  border: none;
+  border-radius: 1000px;
+  color: var(--wxc-join-button-color);
+  cursor: pointer;
+  font-family: $brand-font-regular;
+  font-size: 16px;
+  line-height: 1.5rem;
+  padding: var(--wxc-join-button-padding);
+
+  &:hover {
+    background: var(--wxc-join-button-bg-hover);
+  }
+
+  &:active {
+    background: var(--wxc-join-button-bg-active);
+  }
+
+  &:disabled {
+    background: var(--wxc-join-button-bg-disabled);
+    color: var(--wxc-join-button-color-disabled);
+    cursor: auto;
```

Comment:
doesn't this happen automatically on disabled html buttons?

Reply:
no, probably because I have specified before that the cursor should pe pointer